crypto/build_dependency.sh

```diff
@@ -2,19 +2,34 @@
 
 set -euo pipefail
 
-# relic version or tag
-relic_version="9206ae50"
+PKG_DIR="./"
```

Comment:
we could use the more reliable:
`DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`